frontend/src/components/Shared/CrossingMapPage/CrossingMapPage.js

```diff
@@ -203,6 +232,10 @@ class CrossingMapPage extends Component {
                     toggleShowLongterm={this.toggleShowLongterm}
                     visibleCrossings={visibleCrossings}
                     allCommunities={allCommunities}
+                    openCrossings={openCrossings}
+                    closedCrossings={closedCrossings}
+                    longtermCrossings={longtermCrossings}
+                    cautionCrossings={cautionCrossings}
```

Comment:
Why does `CrossingMapSidebar` need to know about every open/closed/longterm/caution crossing?

I see that we use it to filter out which crossings should show up in the sidebar. 

But would it make more sense to update `visibleCrossings` to actually reflect what's visible?

If we do that, we could even get rid of the `showOpen/showClosed/showCaution/showLongterm` props from `CrossingMap`.